app/src/main/java/com/ex/org/exceptionstest/com/ex/lib/ExceptionReporter.java

```diff
@@ -72,6 +73,7 @@ public static void logException(Exception caughtException){
     }
 
     //TODO: Not thread safe
+    //ERIK - why not thread save? I think only one uncaught exception handler can execute at one time anyways
```

Comment:
well.. this is for caught execptions.. imagine i spin another thread in my app and then call on this method.. not sure how it'll handle things

Reply:
Ah, I see what you mean now. Just make the method synchronized then. Performance is not going to be a factor for this kind of code (I mean, if you enough logged exceptions to cause a slowdown, then something else is probably wrong, haha)

Reply:
makes sense

Reply:
hmm.. actually i don't think that making it block is a good idea... I will leave a comment to add this e to a blockingqueue and release the caller. These might be debug exceptions coming from multiple threads